core/src/com/unciv/logic/civilization/Notification.kt

```diff
@@ -36,6 +37,19 @@ object NotificationIcon {
     const val Barbarians = "ImprovementIcons/Barbarian encampment"
 }
 
+enum class NotificationColor(val color: Color) {
+    Default(Color(1f,1f,1f,1f)),
+    Bombard(Color(0.992f, 0.847f, 0.694f, 1f)),
+    Bad(Color(0.996f, 0.752f, 0.698f, 1f)),
+    LessBad(Color(1f, 0.850f, 0.819f, 1f)),
+    Good(Color(0.745f, 1f, 0.698f, 1f)),
+    LessGood(Color(0.843f, 1f, 0.811f, 1f)),
```

Comment:
I feel like these should be reworded to
`VeryBad`
`Bad`
`VeryGood`
`Good`
To me, very good vs good is a little more clear than good vs less good.

Reply:
I will. Didn't really know what to name these so I just went for less___ :)